avo-hq/avo
# fix: belongs to polymorphic fails to save record

# Description


Fixes https://github.com/avo-hq/avo/issues/666#issuecomment-1046735190

# Checklist:



## Manual review steps


1. Go to a project
1. Open a comment (from the has_many listing)
2. Click edit
3. don't update any fields
4. click save
5. check that you've been redirected to the project page
5. check that the comment is attributed to the same project

Manual reviewer: please leave a comment with output from the test if that's the case.
